src/cli.js

```diff
@@ -236,6 +233,17 @@ Options
                 reporter.isReportServer = boolValue();
                 break;
 
+            case "report-path":
+                if(!value.match((/^(.*?)[\\\/]([^\\\/]+[a-zA-Z])$/) || (/^[^\/\: ]+(\:[0-9]+)?$/))) {
+                    utils.error(`Invalid path report folder. It must be absolute 'path', 'domain' or 'domain:port'.`);
```

Comment:
Improved wording: "It must be an absolute path, domain, or domain:port"

Reply:
I even think that it is not necessarily code to pay attention to the path because there is far too much possibility I propose to accept any type of path without REGEX